src/assets/scss/_links.scss

```diff
@@ -10,11 +10,14 @@ a, button,
   border-radius: 16px;
   border: 0;
 
+  color: #000000;
```

Comment:
seeing any hex code anywhere makes me uneasy. i know you know this but just for posterity: we should eventually make this a var.

Reply:
That's next branch. I'm ripping out just about every color declaration and starting over for theming.